README.md

```diff
@@ -30,6 +30,6 @@
 - Use `_` between a job name and the [job template variables](https://docs.openstack.org/infra/jenkins-job-builder/definition.html#job-template)
 - Use standard capitalization rules, template variables can be an exception to this
 - Examples:
-  - `RPC-AIO_{series}-{context}-{ztrigger}`
-  - `RPC-AIO_master-swift-periodic`
-  - `Merge-Trigger-JJB`
\ No newline at end of file
+  - `RPC-AIO_{series}-{image}-{scenario}-{action}-{ztrigger}`
+  - `RPC-AIO_master-xenial-swift-deploy-periodic`
```

Comment:
It seems simpler to me to then just add more series, for example:

RPC-AIO_kilo2newton141-trusty-swift-leapfrog-periodic
RPC-AIO_liberty2newton141-trusty-swift-leapfrog-periodic